src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts

```diff
@@ -33,7 +33,48 @@ let expectedNotebookContent: nb.INotebookContents = {
 	nbformat: 4,
 	nbformat_minor: 2
 };
+
+let expectedNotebookMarkdownContent: nb.INotebookContents = {
```

Comment:
Generally it's good practice to scope down stuff like this to the suite/test that uses it. Makes it easier to find and relate test artifacts to the things that actually use them. So in these cases since they're each only used by one test I'd just put these in the tests themselves.